src/components/item-display-panel/item-display-panel.component.ts

```diff
@@ -31,7 +31,91 @@ export class ItemDisplayPanelComponent {
         this.onChangeCallback(this._visible);  
     }
 
+    @Input()
+    get boxShadow() {
+        let value: any = this._boxShadow;
+        if (value === 'false') {
+            this._boxShadow = false;
+        }
+        return this._boxShadow;
+    }
+
+    set boxShadow(boxShadow: boolean) {
+        this._boxShadow = boxShadow;
+    }
+
+    @Input()
+    get closeVisible() {
+        let value: any = this._closeVisible;
+        if (value === 'false') {
+            this._closeVisible = false;
+        }
+        return this._closeVisible;
+    }
+
+    set closeVisible(closeVisible: boolean) {
+        this._closeVisible = closeVisible;
+    }
+
+    @Input()
+    get preventClose() {
+        let value: any = this._preventClose;
+        if (value === 'true') {
+            this._preventClose = true;
+        }
+        return this._preventClose;
+    }
+
+    set preventClose(preventClose: boolean) {
+        this._preventClose = preventClose;
+    }
+
+    @Input()
+    get inline() {
+        let value: any = this._inline;
+        if (value === 'true') {
+            this._inline = true;
+        }
+        return this._inline;
+    }
+
+    set inline(inline: boolean) {
+        this._inline = inline;
+    }
+
+    @Input()
+    get animate() {
+        let value: any = this._animate;
+        if (value === 'true') {
+            this._animate = true;
+        }
+        return this._animate;
+    }
+
+    set animate(animate: boolean) {
+        this._animate = animate;
+    }
+
+    @Input()
+    get shadow() {
+        let value: any = this._shadow;
+        if (value === 'true') {
+            this._shadow = true;
+        }
+        return this._shadow;
+    }
+
+    set shadow(shadow: boolean) {
+        this._shadow = shadow;
+    }
+
     private _visible: boolean = false;
+    private _boxShadow: boolean = true;
+    private _closeVisible: boolean = true;
+    private _preventClose: boolean = false;
+    private _inline: boolean = false;
+    private _animate: boolean = false;
+    private _shadow: boolean = false;
 
     // private onTouchedCallback: () => void = () => { };
     private onChangeCallback: (_: boolean) => void = () => { };
```

Comment:
This is only required for ngModel support which this component doesn't use so we can remove it